src/Facility.Core/Http/HttpClientServiceDefaults.cs

```diff
@@ -14,4 +14,9 @@ public sealed class HttpClientServiceDefaults
 	/// The default content serializer.
 	/// </summary>
 	public HttpContentSerializer? ContentSerializer { get; set; }
+
+	/// <summary>
+	/// True to compress all requests by default.
+	/// </summary>
+	public bool? CompressRequests { get; set; }
```

Comment:
```suggestion
	public bool CompressRequests { get; set; }
```
I don't think nullability is an advantage here.

Reply:
I was leaving open the possibility that Facility would turn on compression by default (in a future version) and users could set this to `false` to explicitly opt out.

Can make it non-nullable if you think that's better.

Reply:
Oh... this class constructor could set the default to `true` (in such future scenario)?

Reply:
This is only used by code generation; users cannot set this property.